Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+MAINTAINER Tomasz Janiszewski <janiszt@gmail.com>
```

Comment:
@adamdubiel @vi4m What should be here? See [other Allegro repos](https://github.com/search?l=dockerfile&q=MAINTAINER++user%3Aallegro&ref=searchresults&type=Code&utf8=%E2%9C%93)